pkg/cgroup/resolve_container.go

```diff
@@ -178,6 +175,7 @@ func (c *cache) getPathFromcGroupID(cgroupID uint64) (string, error) {
 
 		getCgroupID, err := utils.GetCgroupIDFromPath(instance.byteOrder, path)
 		if err != nil {
+			instance.cGroupIDToPath.Store(cgroupID, unknownPath)
```

Comment:
@sunya-ch , it seems we want to set cache with key value as cgroupID, unknownPath when error happen, but in previous, the cache is never set, as the code will return before cache setting?

Reply:
@SamYuan1990 we should not store cgroupID and unknownPath here.
This is inside the loop to set cGroupIDToPath values for all traversed path in cgroupPath path.
If the path doesn't provide cgroup (getCgroupID) value, we will just ignore and continue checking the next path (return err in func parameter of WalkDir).